app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  after_save :update_comment_counter
+
+  def update_comment_counter
+    post.increment!(:CommentsCounter)
+  end
```

Comment:
- [ ] OPTIONAL
Following best practices `update_comment_counter` should be marked as private after implementing the `after_save` method.

Reply:
- [ ] This comment has not been addressed yet